oak/common/app_config.h

```diff
@@ -41,6 +46,11 @@ void AddLoggingToConfig(ApplicationConfiguration* config);
 bool AddStorageToConfig(ApplicationConfiguration* config, const std::string& name,
                         const std::string& storage_address);
 
+// Modify application configuration to wire up escrow for the given node.
+// Return value indicates whether given storage node was found.
```

Comment:
Does this require `AddStorageToConfig` to have already been called (and if so does `storage_address` need to match)?  What does `name` identify?